specification/maps/data-plane/Creator/preview/2023-03-01-preview/features.json

```diff
@@ -0,0 +1,1089 @@
+        "operationId": "Features_Put",
+        "x-ms-examples": {
+          "PutFeature": {
+            "$ref": "./examples/features/Features_Put.json"
+          }
+        },
+        "consumes": [
+          "application/geo+json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/FeatureId"
+          },
+          {
+            "$ref": "#/parameters/FeatureBody"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Feature Updated."
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      },
+      "patch": {
+        "description": "This Patch Feature API conforms with the **drafted** [Open Geospatial Consortium API - Features - Part 4: Create, Replace, Update and Delete](http://docs.opengeospatial.org/DRAFTS/20-002.html) document. This API **updates** attributes by performing a merge patch on the feature identified by the provided id in the given collection. It allows for the update of individual feature attributes without implicitly overwriting excluded attributes in the request body.\n\nThe Features API is a part of [Creator](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps), which makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. Features GET APIs follow the [Open Geospatial Consortium API - Features - Part 1: Core corrigendum](http://www.opengis.net/doc/IS/ogcapi-features-1/1.0.1) standard to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v20220901preview/dataset). Datasets consist of multiple feature collections, which are a grouping of [features](https://tools.ietf.org/html/rfc7946#section-3.2) of similar type. This grouping is defined by a common schema referred to as [ontology](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2).\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).",
+        "operationId": "Features_Patch",
```

Comment:
We typically refrain from using the verb as the name of the API.  What about Update?  Seems to match with the description.

Reply:
Thanks again, I've fixed it here as well.